### PR TITLE
Adding artificial sweetener and spawning it around the world.

### PR DESCRIPTION
**Pull request by Lamandus**
#### Summary
Summary: Content "Adding artificial sweetener powder"

#### Purpose of change
Adding artificial sweetener to the game. Right now no reciepes will use it, I will commit it as a seperate PR after my tea and coffee gets merged. I need to check which reciepes can use artificial sweetener aka aspartam.

#### Describe alternatives you've considered
keep it diet free.

#### Testing
worked for my game at least.

#### Additional context
There are medical controvercies about artificial sweetener being harmful or not. I will give it the benefit of the doubt and leave it on the same level as sugar (with no calories).

### CODE REVIEW

**Comment by CountAlex** [comment]
I think it would be better to add it to few recipes where it can be used for sweet flavor rather than other sugar properties (like fermenting or preservating). Otherwise it'll be pretty useless.

**Comment by Lamandus** [comment]
> 
> 
> I think it would be better to add it to few recipes where it can be used for sweet flavor rather than other sugar properties (like fermenting or preservating). Otherwise it'll be pretty useless.

https://github.com/CleverRaven/Cataclysm-DDA/pull/40955 Way ahead of you :)

File: data/json/items/comestibles/spice.json [hunk 1]
```diff
@@ -154,5 +154,13 @@
     "name": { "str_sp": "mustard powder" },
     "description": "A fragnant yellow powder.  Not edible in this form.",
     "color": "red"
+  },
+  {
+    "id": "artificial_sweetener",
+    "copy-from": "sugar",
+    "type": "COMESTIBLE",
+    "name": "artificial sweetener",
+    "description": "Sweet, sweet sugar? No, it is bitter-sweet artificial sweetener.  No calories, no worries.",
```
Comment (AMurkin) [inline data/json/items/comestibles/spice.json:163]: And change the summary line to this:
Summary: Content "Adding artificial sweetener powder"
```suggestion
    "description": "Sweet, sweet sugar?  No, it is bitter-sweet artificial sweetener.  No calories, no worries.",
```

Comment (Lamandus) [reply data/json/items/comestibles/spice.json]: done, thank you for your time

**Pull request merged**